package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-toastr",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
@knownasilya  beside that you said to put brocoli-funnel and broccoli-merge-trees as devDependencies I am still for putting them as dependecies. Its your call.